### PR TITLE
Update documentation on ratios to include floor and ceil operations

### LINKED ISSUE #552: Update Ratio documentation
It looks like documentation has not been updated for the latest Ratio changes, from https://github.com/Agoric/agoric-sdk/commit/dc8d6dca5898890ef4d956c83685bc28eb189791#diff-fe679b1c51464cc0f59ad0c778d5c879cf52dfcb78635b8a2f263f5ea57ca3dc on. 

The file that needs updating is main/zoe/api/contract-support/ratio-math.md.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes #552

### CODE REVIEW

File: main/zoe/api/contract-support/ratio-math.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+denominator to cancel it out; e.g. 5 gallons * (10 miles / 1 gallon) = 50
+miles. Similarly, dividing an amount by a ratio returns a result amount with the
+denominator's brand, so the amount should be the same brand as the numerator to
+cancel it out.
+
+In order to support precision calculations, the multiplication and division
+operations require that the caller specify whether the result should be rounded
+up or down. These operations produce Amounts, so they have to end by converting
+the ratio to an integer with a division. This may require roundoff, and correct
+calculations requrie that the caller choose which is appropriate.
```
Comment (katelynsills) [inline main/zoe/api/contract-support/ratio-math.md:29]: Small typo, text reads well though
```suggestion
calculations require that the caller choose which is appropriate.
```

File: main/zoe/api/contract-support/ratio-math.md [hunk 2]
```diff
@@ -80,23 +88,24 @@ Throws messages for errors:
...
-   not change.
-   
-The above describes what is done to determine the result, not precisely how it's
-done in the source code. 
+3. If that results in an integer, that value is returned, otherwise, the value
+  is rounded down (for floorMultiplyBy) or up (for ceilMultiplyBy) to the next
+  integer.
+
+  The above describes what is done to determine the result, not precisely how
+  it's done in the source code.
```
Comment (katelynsills) [inline main/zoe/api/contract-support/ratio-math.md:108]: I realize this comment isn't new, but it seems strange to me that we can't describe how it's done in the source code. A user may want to know

File: main/zoe/api/contract-support/ratio-math.md [hunk 3]
```diff
@@ -119,21 +128,24 @@ const Dollars47 = AmountMath.make(dollarBrand, 47n);
...
 Returns an immutable `Amount`.  Its brand is the `ratio`'s *denominator*'s brand.
+
 Its value is determined by:
 1. Multiplying the `amount` value by the `ratio`'s denominator's value.
 2. Dividing the result from step 1 by the `ratio`'s numerator's value.
-3. Applying a floor to the result from step 2 to round it down to
-    the nearest integer. If the step 2 result is already an integer, its value does
-    not change.
+3. If that results in an integer, that value is returned, otherwise, the value
+  is rounded down (for floorMultiplyBy) or up (for ceilMultiplyBy) to the next
```
Comment (katelynsills) [inline main/zoe/api/contract-support/ratio-math.md:143]:
```suggestion
  is rounded down (for floorDivideBy) or up (for ceilDivideBy) to the next
```

**Review by katelynsills — APPROVED** [review]
Looks good, some small requests for changes

File: main/zoe/api/contract-support/ratio-math.md (hunk 2 above)
Comment (Chris-Hibbert) [reply main/zoe/api/contract-support/ratio-math.md]: The step-by-step does describe the source code. I think the old version was vaguer, and that's the reason the caveat was there originally. I've dropped the caveat.

File: main/zoe/api/contract-support/ratio-math.md (hunk 3 above)
Comment (Chris-Hibbert) [reply main/zoe/api/contract-support/ratio-math.md]: oops.  Fixed.

**Pull request merged**